docs/tutorials/client-server.md

```diff
@@ -0,0 +1,230 @@
+
+- Add a [`Skip`](xref:Bonsai.Reactive.Skip) operator after the [`Router`](xref:Bonsai.ZeroMQ.Router) in a separate branch and connect this to a [`PublishSubject`](xref:Bonsai.Reactive.PublishSubject). 
+- Set the [`Skip`](xref:Bonsai.Reactive.Skip) operator's `Count` property is set to 1, and name the [`PublishSubject`](xref:Bonsai.Reactive.PublishSubject) 'NextMessage'.
+
+:::workflow
+![Server next message](~/workflows/server-next-message.bonsai)
+:::
+
+The logic here is that we use [`Skip`](xref:Bonsai.Reactive.Skip) to create a sequence that lags exactly 1 message behind the [`Router`](xref:Bonsai.ZeroMQ.Router) sequence of received messages, i.e. when the first message is received, `NextMessage` will not produce a result until the second message is received. We can then use this inside our [`SelectMany`](xref:Bonsai.Reactive.SelectMany) logic for generating server messages. 
+
+- Add a [`SelectMany`](xref:Bonsai.Reactive.SelectMany) operator after the [`Router`](xref:Bonsai.ZeroMQ.Router) in a separate branch and name it ‘SelectAllClients’.
+- Inside the [`SelectMany`](xref:Bonsai.Reactive.SelectMany) operator, create 2 [`SubscribeSubject`](xref:Bonsai.Expressions.SubscribeSubject) operators and link them to the `ClientAddresses` and `NextMessage` subjects. 
+- Connect the `ClientAddresses` subscription to the workflow output via a [`TakeUntil`](xref:Bonsai.Reactive.TakeUntil) operator and use `NextMessage` as the second input. 
+
+Now, our `SelectAllClients` will produce a sequence of all unique client addresses every time the server receives a message. Connect the output of `SelectAllClients` to a [`WithLatestFrom`](xref:Bonsai.Reactive.WithLatestFrom) with the [`Router`](xref:Bonsai.ZeroMQ.Router) as its second input. In this context [`WithLatestFrom`](xref:Bonsai.Reactive.WithLatestFrom) combines each client address from `SelectAllClients` with the most recent received message. The result is that when a message is received from the client, we produce several copies of the message 'addressed' to each connected client.
+
+:::workflow
+![Select all clients and package message](~/workflows/format-select-all-clients.bonsai)
```

Comment:
Seems like this workflow has a disabled or invalid `BounceBack` node which perhaps should be removed to avoid confusion if not needed?